user-guide/gpu.rst

```diff
@@ -118,11 +117,10 @@ code.
 Submitting jobs to the GPU nodes
 --------------------------------
 
-Two additional options are needed in GPU job submission scripts over those in standard jobs:
+An additional option is needed in GPU job submission scripts over those in standard jobs:
 
- * ``-q gpu`` This option is required to submit the job to the ``gpu`` queue on Cirrus
- * ``ngpus=N`` (where ``N`` is the number of GPU accelerators you wish to use). This resource 
-   request needs to be added to your ``select`` statement
+ * ``--gres=gpu:N`` (where ``N`` is the number of GPU accelerators you wish to use). This resource 
+   request needs to be added to your Slurm script.
 
 .. note:: We generally recommend that you should request 10 CPU cores per GPU accelerator even if you do not need them.
```

Comment:
I think this is wrong now. You automatically get assigned 10 cores and 1/4 of RAM for each GPU you request.

Reply:
yes, I'll re-write this